modin/pandas/test/test_dataframe.py

```diff
@@ -2390,15 +2390,29 @@ def test_interpolate(self):
         with pytest.warns(UserWarning):
             pd.DataFrame(data).interpolate()
 
-    def test_kurt(self):
-        data = test_data_values[0]
-        with pytest.warns(UserWarning):
-            pd.DataFrame(data).kurt()
-
-    def test_kurtosis(self):
+    @pytest.mark.parametrize("axis", axis_values, ids=axis_keys)
+    @pytest.mark.parametrize("skipna", bool_arg_values, ids=bool_arg_keys)
+    @pytest.mark.parametrize("level", [None, -1, 0, 1])
+    @pytest.mark.parametrize("numeric_only", bool_arg_values, ids=bool_arg_keys)
+    @pytest.mark.parametrize("method", ["kurtosis", "kurt"])
+    def test_kurt_kurtosis(self, axis, skipna, level, numeric_only, method):
+        # FIXME remove level condition
```

Comment:
Is `level` not working above?

Reply:
Yes. Reason not established.

For example, you can see log.
<details> <summary>log </summary>

```shell
______________________________________ TestDataFrameDefault.test_kurt_kurtosis[kurt-None-0-None-over_columns_str] _______________________________________

self = <modin.pandas.test.test_dataframe.TestDataFrameDefault object at 0x000001A6A78BA5B0>, axis = 'columns', skipna = None, level = 0
numeric_only = None, method = 'kurt'

    @pytest.mark.parametrize("axis", axis_values, ids=axis_keys)
    @pytest.mark.parametrize("skipna", bool_arg_values, ids=bool_arg_keys)
    @pytest.mark.parametrize("level", [None, -1, 0, 1])
    @pytest.mark.parametrize("numeric_only", bool_arg_values, ids=bool_arg_keys)
    @pytest.mark.parametrize("method", ["kurtosis", "kurt"])
    def test_kurt_kurtosis(self, axis, skipna, level, numeric_only, method):
        # FIXME remove level condition
        #if level in [-1, 0]:
        #    return
        data = test_data_values[0]
        modin_df, pandas_df = pd.DataFrame(data), pandas.DataFrame(data)
        try:
            pandas_result = getattr(pandas_df, method)(
                axis, skipna, level, numeric_only
            )
        except Exception as e:
            with pytest.raises(type(e)):
                repr(
                    getattr(modin_df, method)(axis, skipna, level, numeric_only)
                )  # repr to force materialization
        else:
            modin_result = getattr(modin_df, method)(axis, skipna, level, numeric_only)
            print(modin_result, pandas_result)
>           df_equals(modin_result, pandas_result)

modin\pandas\test\test_dataframe.py:2416:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
modin\pandas\test\utils.py:457: in df_equals
    if df1 != df2:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self =         0     1     2     3     4     5     6     7     8     9    10  ...  col58  col59  col6  col60  col61  col62  c...rue  True  True  ...   True   True  True   True   True   True   True   True  True  True  True

[256 rows x 320 columns]

    def __nonzero__(self):
>       raise ValueError(
            f"The truth value of a {type(self).__name__} is ambiguous. "
            "Use a.empty, a.bool(), a.item(), a.any() or a.all()."
        )
E       ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().

..\..\..\appdata\local\continuum\miniconda3\envs\modin\lib\site-packages\pandas\core\generic.py:1478: ValueError
```
</details>

Reply:
I have seen this issue before, the type of `modin_result` does not match `pandas_result`. One is a `Series` the other is a `DataFrame`. I will see if I can fix.

Reply:
For now, let's default to pandas on `level`. The result was not correct when `level` was set before, so we should default to pandas implementation until it is fully implemented.

Then you can re-enable the tests

Reply:
done